src/OpenTelemetry/Internal/MathHelper.cs

```diff
@@ -15,12 +15,16 @@
 // </copyright>
 
 using System.Diagnostics;
+#if NETCOREAPP3_0_OR_GREATER
+using System.Numerics;
+#endif
 using System.Runtime.CompilerServices;
 
 namespace OpenTelemetry.Internal;
 
 internal static class MathHelper
 {
+#if !NETCOREAPP3_0_OR_GREATER
```

Comment:
Maybe leave this as is and also keep the test for 8/16/32 bits?

Reply:
Ok, removed this line and added the 8/16/32 bits version back.